traefik/templates/ingressclass.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
     ingressclass.kubernetes.io/is-default-class: {{ .Values.ingressClass.isDefaultClass | quote }}
   labels:
     {{- include "traefik.labels" . | nindent 4 }}
-  name: {{ template "traefik.fullname" . }}
+  name: {{ .Values.ingressClass.name }}
```

Comment:
This will break backward compatibility for users who deploy chart under non-default release name, which is common use case and should be taken into account here.

Reply:
now, with commit from @darkweaver87 , it won't break.